lib/tools/apk-utils.js

```diff
@@ -152,7 +164,8 @@ apkUtilsMethods.startApp = async function startApp (startAppOptions = {}) {
     }
     return stdout;
   } catch (e) {
-    throw new Error(`Cannot start the '${startAppOptions.pkg}' application. ` +
+    const appDescriptor = startAppOptions.pkg || `${startAppOptions.action} ${startAppOptions.optionalIntentArguments}`;
+    throw new Error(`Cannot start the '${appDescriptor}' application. ` +
       `Visit ${ACTIVITIES_TROUBLESHOOTING_LINK} for troubleshooting. ` +
```

Comment:
I think it would also be useful to update the troubleshooting document with some information about intents